js/main.js

```diff
@@ -0,0 +1,54 @@
+var $pokemonList = document.querySelector('.pokemon-list');
+
+function getPokemonDataAll2(datagen) {
```

Comment:
Why the `2`?

Reply:
i was testing the function to see if it worked, forgot i had a placeholder name.